### PR TITLE
TG 318 321 | Network node proponent fields

### PR DESCRIPTION
**Pull request by diogo-wopta**

### CODE REVIEW

File: document/fpdf-utils.go [hunk 1]
```diff
@@ -28,6 +28,13 @@ const (
 	proposal         = "PROPOSTA"
 )
 
+var productCompanyMap map[string]string = map[string]string{
+	models.LifeProduct:    "AXA FRANCE VIE S.A.",
+	models.GapProduct:     "Sogessur SA",
+	models.PmiProduct:     "Global Assistance",
+	models.PersonaProduct: "Global Assistance",
+}
```
Comment (Yousef-Hammar) [inline document/fpdf-utils.go]: Should we use ` company` as key for this map?

File: models/network-node.go [hunk 2]
```diff
@@ -61,6 +62,8 @@ type AgencyNode struct {
 	RuiRegistration    time.Time             `json:"ruiRegistration" firestore:"ruiRegistration" bigquery:"-"`
 	BigRuiRegistration bigquery.NullDateTime `json:"-" firestore:"-" bigquery:"ruiRegistration"`
 	Skin               *Skin                 `json:"skin,omitempty" firestore:"skin,omitempty" bigquery:"-"`
+	Pec                string                `json:"pec,omitempty" firestore:"pec,omitempty" bigquery:"pec"`
+	Website            string                `json:"website,omitempty" firestore:"website,omitempty" bigquery:"website"`
```
Comment (Yousef-Hammar) [inline models/network-node.go]: do we need to save these fields is a column in bigquery tables?

Comment (diogo-wopta) [reply models/network-node.go]: I'm all for saving everything in big query to avoid missing fields for future queries/analysis

Comment (diogo-wopta) [reply models/network-node.go]: Deleted the fields from bigquery

**Review by Yousef-Hammar — APPROVED** [review]

**Pull request merged**